package.json

```
@@ -0,0 +1,43 @@
+{
+  "name": "cobda-web",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.11.4",
+    "@testing-library/react": "^11.1.0",
+    "@testing-library/user-event": "^12.1.10",
+    "@types/jest": "^26.0.15",
+    "@types/node": "^12.0.0",
+    "@types/react": "^16.9.53",
+    "@types/react-dom": "^16.9.8",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-scripts": "4.0.1",
```

Apologize for late question, do you think this dependency has to keep updated too? I'm not quite sure but please recheck.
```suggestion
    "react-scripts": "^4.0.1",
```

---

FYI, in future since the backend code will be put in here too, we might not use `react-scripts` so you can just keep it like this first. But it's still good to check 🧐

---

Since `react-scripts` version `4.0.1` is already the latest one, this will work perfectly fine for now and soon. However, I agreed to keep it updated for more flexibility.